### DISCUSSION TITLE
Welcome to WiFi-Widget Discussions!

### DISCUSSION THREAD

**Discussion by w2sv** [w2sv/WiFi-Widget#111] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by cengizguler998-ux** [comment]
Türk Telekom İnternet çekmiyor Türkiye gibi yerde çekmiyor Readar P 13 kulan yorum ev wifi kulan yorum ve sıkıntı yan odan çekim yor

**Comment by cengizguler998-ux** [comment]
Telefonu da P13 Reader intrenet çekim yor sıkıntı lı yardımcı olum sunu sevinirim minetar kalırım size saygılarımı teşekkür ederim

**Comment by cengizguler998-ux** [comment]
Türk Telekom İnternet çekmiyor
